sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java

```diff
@@ -28,9 +28,17 @@
  * Class containing code snippets that will be injected to README.md.
  */
 public class ReadmeSamples {
-    private final KeyVaultAccessControlClient keyVaultAccessControlClient = new KeyVaultAccessControlClient(null);
+    private final KeyVaultAccessControlClient keyVaultAccessControlClient =
```

Comment:
If you update the code snippet samples in README, you also need to regenerate it so it matches the contents in the README.

https://github.com/Azure/azure-sdk-for-java/wiki/Building#code-snippets-in-readme

Reply:
@hongli750210 This means going to the README file and updating the line numbers in every code snippet to match the ones in this file.

Reply:
Fixed in the new version.